### PR TITLE
Migrated `networkingip` datasource to Framework

### PR DESCRIPTION
**Pull request by ezilber-akamai**
## 📝 Description

Migrated `networkingip` datasource to Framework

## ✔️ How to Test

`make PKG_NAME="linode/networkingip" testacc`

Ticket: TPT-1991

### CODE REVIEW

**Comment by ezilber-akamai** [comment]
/acctest sha=1e36ecdd8e12bce73b6760c716a5d712e43d25d2 pkg=linode/networkingip

**Review by zliang-akamai — APPROVED** [review]
Nice work!

File: linode/networkingip/framework_datasource.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+) {
+	resp.Schema = frameworkDatasourceSchema
+}
+
+func (d *DataSource) Read(
+	ctx context.Context,
+	req datasource.ReadRequest,
+	resp *datasource.ReadResponse,
+) {
+	client := d.client
```
[the remark this answers is not in the document]
Comment (ezilber-akamai) [reply linode/networkingip/framework_datasource.go]: Fixed!

**Comment by ezilber-akamai** [comment]
/acctest sha=2cbacab2b0014211f82fb84101dd0f6476d4720c pkg=linode/networkingip

**Comment by ezilber-akamai** [comment]
/acctest sha=e58a8fb4eee149c7b27e157ba843a541b936bb74 pkg=linode/networkingip

**Pull request merged**